### DISCUSSION TITLE
Very nice music player, will be contributing to making less energy intensive

### DISCUSSION THREAD

**Discussion by ghost** [basharovV/musicat#18] [General]
The program is fantastic, with a few bugs - I will try to help where i can to fix bugs and enhance the UI

Right now there are some issues with energy consumption, it's a hungry app!

I'm also going to fork and create a separate app aimed to be more artist centric towards distribution

**Comment by basharovV** [comment]
Hi! Thanks for contributing and raising issues, stability is the main thing this app needs before 1.0. Any specific examples of hungriness? Is it during playback, import, or just in idle mode?

**Reply by ghost** [reply]
It seems to use a bit more than safari does during idle and then during playback it becomes quite hungry

IDLE:
<img width="942" alt="Screenshot 2024-10-03 at 9 28 02 AM" src="https://github.com/user-attachments/assets/f47a9599-288e-4f65-bce3-14ea2c600836">

PLAYBACK:
<img width="937" alt="Screenshot 2024-10-03 at 9 28 27 AM" src="https://github.com/user-attachments/assets/1ed82e2f-f3b9-4d72-ba93-d194f785d2ec">

**Reply by basharovV** [reply]
Relevant - I just fixed a memory leak issue that resulted in memory climbing around 5MB/s when idle, so that might have been what you encountered. Idle RAM usage should be around ~ 50MB now.
